2021-09-27 17:29:50.287 | ERROR    | core.server:request_validation_exception_handler:184 - 请求参数格式错误
URL:POSThttp://10.2.15.74:8010/login/access-token
Headers:Headers({'host': '10.2.15.74:8010', 'connection': 'keep-alive', 'content-length': '53', 'accept': 'application/json', 'user-agent': 'Mozilla/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.159 Safari/537.36', 'content-type': 'application/json', 'origin': 'http://10.2.15.74:8010', 'referer': 'http://10.2.15.74:8010/api/docs', 'accept-encoding': 'gzip, deflate', 'accept-language': 'zh-CN,zh;q=0.9'})
Traceback (most recent call last):
  File "D:\Python\lib\site-packages\starlette\exceptions.py", line 71, in __call__
    await self.app(scope, receive, sender)
  File "D:\Python\lib\site-packages\starlette\routing.py", line 580, in __call__
    await route.handle(scope, receive, send)
  File "D:\Python\lib\site-packages\starlette\routing.py", line 241, in handle
    await self.app(scope, receive, send)
  File "D:\Python\lib\site-packages\starlette\routing.py", line 52, in app
    response = await func(request)
  File "D:\Python\lib\site-packages\fastapi\routing.py", line 217, in app
    raise RequestValidationError(errors, body=body)
fastapi.exceptions.RequestValidationError: 1 validation error for Request
body -> password
  field required (type=value_error.missing)

